packages/cli/src/lib/control-plane-service.ts

```diff
@@ -1,16 +1,26 @@
 // @ts-ignore it's ok if CPS isn't available
-import ControlPlaneService from '@segment/control-plane-service-client'
+import type { Client } from '@segment/control-plane-service-client'
 // @ts-ignore it's ok if CPS isn't available
-export * from '@segment/control-plane-service-client'
+export type * from '@segment/control-plane-service-client'
 
-export const controlPlaneService = new ControlPlaneService({
-  name: 'control-plane-service',
-  url: 'http://control-plane-service.segment.local',
-  userAgent: 'Segment (actions cli)',
-  timeout: 10000,
-  headers: {
-    // All calls from this script are system-to-system and shouldn't require authz checks
-    // TODO remove this once we support auth tokens
-    'skip-authz': '1'
+let client: Client
+
+export default async function loadCPS(): Client {
+  if (!client) {
+    const module = await import('@segment/control-plane-service-client')
```

Comment:
Instead of dynamic import, can we use `require`?
```suggestion
export default function createClient(): Client {
  if (!client) {
    const module = require('@segment/control-plane-service-client')
```

Reply:
You may need an eslint ignore for the `require` but there should be some prior art somewhere:
https://github.com/segmentio/action-destinations/blob/df68122c34cfcac6f8cabb28cd6ea64f3b57bbf5/packages/cli/src/lib/destinations.ts#L21